src/accountregdlg.h

```diff
@@ -1,6 +1,7 @@
 /*
  * accountregdlg.h
- * Copyright (C) 2001, 2002, 2006  Justin Karneges, Remko Troncon
+ * Copyright (C) 2001-2019  Psi Team
```

Comment:
I personally don't like this bloated copyright stuff. Look at the line below, it's reflects the history good enough.
And yes some real code logic changes were made in 2017, not 2019.

Reply:
It is not a problem.
And I have edited this file in 2019 for this perfect change :)